src/db.rs

```diff
@@ -0,0 +1,52 @@
+use tokio_postgres::{NoTls, Error};
+use dotenv::dotenv;
+use std::env;
+
+pub async fn insert_user(name: &str, age: i32) -> Result<(), Error> {
+    dotenv().ok();
+    let database_url = env::var("DATABASE_URL").expect("DATABASE_URL must be set");
```

Comment:
This too, probably should be attached to some global/pseudo-global context that is available readily.